src/site/_data/i18n/paths/lighthouse_accessibility.yml

```diff
@@ -26,6 +26,9 @@ overview:
   zh: 这些检查能够突出可以改进您的 Web 应用无障碍功能的机会。只有一部分无障碍功能问题可以被自动检测到，所以我们也鼓励您进行人工测试。
 
 topics:
+  a11y_overview:
+    en: Accessibility Overview
+
```

Comment:
How does the i18n pipeline work? Does this get updated later?

Reply:
We request new translations in batches. Until the next batch is rganized and contracted, it will fall back to English string.